spec/octokit/client/apps_spec.rb

```diff
@@ -278,6 +278,13 @@
       end
     end # .delete_installation
 
+    describe '.list_app_hook_deliveries' do
+      it 'lists hook deliveries for an app' do
+        response = @jwt_client.list_app_hook_deliveries
+        expect(response).to be_kind_of(Array)
```

Comment:
What do you think about expanding the list of assertions here to include a total count check?

Reply:
Sure, that sounds good to me. I added an assertion that the returned count should be 30, the default per-page amount.

I've got to admit that I'm still a bit confused about how to successfully record a new VCR cassette, which is needed for the tests. Do I need to provide some of the ENV variables listed in [the table](https://github.com/octokit/octokit.rb#running-and-writing-new-tests) in the README and run the specs with those variables?